api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -46,6 +47,16 @@
 import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.core.UriInfo;
 
+import com.google.common.base.Function;
```

Comment:
It's been agreed to follow an import order. Can you update your IDE settings and the PR to get rid of the import order changes? Thanks